backend/app/schemas.py

```
@@ -199,9 +234,14 @@
 #     content: str
 
 
+
 class ChatRequest(BaseModel):
     messages: List[str]
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)